test/test_playbooks/composite_content_view.yml

```diff
@@ -0,0 +1,37 @@
+---
+- hosts: localhost
```

Comment:
You need to restructure the playbooks. Take a look at i.e. the operating_system playbook. Also you might want to reread the README. ;)
By now, the tests are just not run, because localhost is not in the inventory.